Gemfile

```diff
@@ -117,7 +117,7 @@ group :test do
   gem "selenium-webdriver"
   gem "shoulda-matchers"
   gem "simplecov"
-  gem "vcr"
+  gem "vcr", github: "vcr/vcr", ref: "7ac8292c289ca98dcb4254b59e1ee29e2205d8b3"
```

Comment:
Can you add a comment (e.g. `# fixes a syntax error bug between VCR and Webmock`) to explain why it's pinned?

Reply:
This is in the git blame, but sure.